Profile plot refac

agenius-logesh opened the pull request:
Removed parameters.
Colors matched according to the group levels.
gynSymCol changed to gynNameCol.
foldChangeLines changed to foldChangeThreshold and log2() is entered inside the code.
DGEdata input changed to DGEobj.
For ggplot no gynSymLables - removed.

saranya-ag committed:
2021 03 refactor obsplot 2 (#29)

agenius-logesh committed:
Replaced asserthat_see_that() to any().

agenius-logesh committed:
Revert Back to original.

agenius-logesh committed:
Merge remote-tracking branch 'DGEobj.plots-upstream/develop' into develop

agenius-logesh committed:
Merge remote-tracking branch 'DGEobj.plots_MA-upstream/Finalize-Profile-Plot' into profile-plot-refac

agenius-logesh committed:
profileplot refactord.

agenius-mohammed-ali commented on R/profilePlot.R:
```diff
@@ -134,20 +123,28 @@ profilePlot <- function(contrastDF,
+    if (!missing(geneNameCol)) {
+        assertthat::assert_that(!is.null(geneNameCol),
+                                length(geneNameCol) == 1,
+                                geneNameCol %in% names(DGEobj::getType(dgeObj, type = "geneData")[[1]]),
```
- There must be check for the existence of geneData in the passed DGEobj
- There must be check for the geneData length

agenius-logesh replied:
done.

agenius-mohammed-ali commented on R/profilePlot.R:
```diff
@@ -337,28 +273,29 @@ profilePlot <- function(contrastDF,
-            var.annot$Row.names <- NULL
+        if (!missing(geneNameCol)) {
+            gene_data <- DGEobj::getItem(dgeObj, "geneData") %>%
+                dplyr::select(all_of(geneNameCol))
```
- It is one column so why using `all_of` function.
- missing `dplyr::` before `all_of`

agenius-logesh replied:
It makes a waring message if we not using all_of()

agenius-mohammed-ali commented on R/profilePlot.R:
```diff
@@ -246,36 +195,13 @@ profilePlot <- function(contrastDF,
+
     # Columns to plot
     # Capture the labels from the colname
     xlabel <- make.names(logIntCol)
```
why not using make names on all column names one time?

agenius-logesh replied:
this is did by you only so I cant get your point mohammed.

agenius-logesh committed:
changes done.

agenius-logesh committed:
example changed.

agenius-logesh committed:
Final update.

agenius-logesh committed:
profilePlot final Updated.

saranya-ag committed:
2021 07 refactor map dg eobj (#32)

merging as WIP

agenius-logesh committed:
Refactoring of logRatio plot (#34)

merging as WIP

agenius-logesh committed:
2021 07 volcano refac (#35)

merging as WIP

saranya-ag committed:
2021 07 refactor all plots copy (#36)

merging as WIP

agenius-logesh committed:
import function changed.

agenius-logesh committed:
topTables and geneData check and test cases added.

agenius-logesh committed:
merge back from develop.

Merge remote-tracking branch 'DGEobj.plots-upstream/develop' into profile-plot-refac

# Conflicts:
#	R/plotDispersion.R
